### DISCUSSION TITLE
Edit dropdown menu

### DISCUSSION THREAD

**Discussion by janlin002** [JedWatson/react-select#5593] [Q&A]
hi everyone!
How can I edit the input value without having to re-enter it?
<img width="594" alt="截圖 2023-03-25 下午1 23 15" src="https://user-images.githubusercontent.com/68535430/227698054-90c18741-9fd8-4e27-8a03-e1d553b7798e.png">
For example, I want to search for "Oceans," but when I type "s" the input box clears.

**Comment by manjushsh** [comment]
Hi @janlin002,
Here is [Example Sandbox](https://codesandbox.io/s/single-keep-val-in-select-fjwst1?file=/example.tsx) using `inputValue` and `onInputChange` for requirement

Is this expected output?

**Comment by janlin002** [comment]
Thank you, that's exactly what I needed.
